rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2023,6 +2030,14 @@ protected Integer commitFederation(boolean dryRun, Keccak256 hash) throws IOExce
                 activations
             )
         );
```

Comment:
Maybe also log the old and new federation addresses

Reply:
Done. https://github.com/rsksmart/rskj/blob/9575ad1bda1683127997608758966a132c73be7b/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L2059